python/tvm/relay/frontend/tensorflow2_ops.py

```diff
@@ -0,0 +1,179 @@
+    def _impl(inputs, attr, params, prelude):
+        dtype_str = attr["element_dtype"].name
+        input_shape = get_tensor_array_shape(inputs[0], dtype_str, prelude)
+
+        if input_shape is None:
+            read_func = prelude.get_global_var("tensor_array_read", dtype_str)
+            out = read_func(inputs[0], _op.take(inputs[1], tvm.relay.const(0)))
+        else:
+            static_tensor_array_ops = StaticTensorArrayOps(prelude, dtype_str, input_shape)
+            static_tensor_array_ops.register()
+            read_func = static_tensor_array_ops.get_global_var("tensor_array_read")
+            out_tensor = read_func(inputs[0], _op.take(inputs[1], tvm.relay.const(0)))
+            get_data_func = static_tensor_array_ops.get_global_var("tensor_get_data")
+            out = get_data_func(out_tensor)
+        return out
+
+    return _impl
+
+
+def _tensorlist_stack():
+    def _impl(inputs, attr, params, prelude):
+        dtype_str = attr["element_dtype"].name
+        input_ta_shape = get_tensor_array_shape(inputs[0], dtype_str, prelude)
+
+        if input_ta_shape is None:
+            stack_func = prelude.get_global_var("tensor_array_stack", dtype_str)
+            out = stack_func(inputs[0])
+        else:
+            static_tensor_array_ops = StaticTensorArrayOps(prelude, dtype_str, input_ta_shape)
+            static_tensor_array_ops.register()
+            stack_func = prelude.get_global_var_static(
+                "tensor_array_stack", dtype_str, input_ta_shape
+            )
+            out_tensor = stack_func(inputs[0])
+            out_shape = (Any(),) + input_ta_shape
+            static_tensor_array_ops = StaticTensorArrayOps(prelude, dtype_str, out_shape)
+            static_tensor_array_ops.register()
+            get_data_func = prelude.get_global_var_static("tensor_get_data", dtype_str, out_shape)
+            out = get_data_func(out_tensor)
+
+        return out
+
+    return _impl
+
+
+def _tensorlist_from_tensor():
+    def _impl(inputs, attr, params, prelude):
+        dtype_str = attr["element_dtype"].name
+        input_ta_shape = _infer_type_with_prelude(inputs[0], prelude).shape
+
+        if input_ta_shape is None:
+            unstack_func = prelude.get_global_var("tensor_array_unstack", dtype_str)
+            out = unstack_func(inputs[0])
+        else:
+            static_tensor_array_ops = StaticTensorArrayOps(prelude, dtype_str, input_ta_shape)
+            static_tensor_array_ops.register()
+            unstack_func = prelude.get_global_var_static(
+                "tensor_array_unstack", dtype_str, input_ta_shape
+            )
+            out = unstack_func(inputs[0])
+        return out
+
+    return _impl
+
+
+_convert_map = {
+    "TensorListFromTensor": _tensorlist_from_tensor(),
+    "TensorListGetItem": _tensorlist_get_item(),
+    "TensorListReserve": _tensorlist_reserve(),
+    "TensorListSetItem": _tensorlist_set_item(),
+    "TensorListStack": _tensorlist_stack(),
```

Comment:
seems there are several other TensorList related ops like TensorListScatter are not added here, what are the reasons you chose the list here to support?

Reply:
The reason why I chose these operators is due to the models I am working on only contain these ops, It will be ongoing efforts for adding new operator support